### DISCUSSION TITLE
Telefe Bahía Blanca

### DISCUSSION THREAD

**Discussion by GiovanniSebastian** [iptv-org/community#865] [Q&A]
### Channel ID

TelefeBahiaBlanca.ar

### Stream URL

https://bahia.mitelefe.com/vivo

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19840
